analytics/lib/time_utils.py

```diff
@@ -8,7 +8,7 @@
 # If min_length is greater than 0, pads the list to the left.
 # So informally, time_range(Sep 20, Sep 22, day, None) returns [Sep 20, Sep 21, Sep 22],
 # and time_range(Sep 20, Sep 22, day, 5) returns [Sep 18, Sep 19, Sep 20, Sep 21, Sep 22]
-def time_range(start, end, frequency, min_length):
+def time_range(start, end, frequency):
```

Comment:
Did this pass the tests? http://zulip.readthedocs.io/en/latest/testing.html might be helpful. 
In particular this may have a mypy failure.